elements/pfe-autocomplete/src/pfe-autocomplete.js

```diff
@@ -204,6 +202,43 @@ class PfeAutocomplete extends PFElement {
     this.setAttribute("debounce", val);
   }
 
+  _slotchangeHandler() {
+    // input box
+    let slotNodes = this.shadowRoot.querySelector("slot").assignedNodes();
+    let slotElems = slotNodes.filter(n => n.nodeType === Node.ELEMENT_NODE);
+
+    if (slotElems.length === 0) {
+      console.warn(
+        `${PfeAutocomplete.tag}: There must be a input tag in the light DOM`
+      );
+
+      return;
+    }
+
+    this._input = slotElems[0];
+
+    if (this._input.tagName.toLowerCase() !== "input") {
+      console.warn(
+        `${PfeAutocomplete.tag}: The only child in the light DOM must be an input tag`
+      );
+
+      return;
+    }
+
+    this._input.addEventListener("input", this._inputChanged.bind(this));
+    this._input.addEventListener("blur", this._closeDroplist.bind(this));
+
+    this._input.setAttribute("role", "combobox");
+    this._input.setAttribute("aria-label", "Search");
```

Comment:
Hardcoding this means it can't be translated - can we add an attribute to pass in this content?

Reply:
@castastrophe - I think we should accept an aria-label if one is already set and set "Search" as a fallback if one is not provided. Would that work for you?

Reply:
Sure!  Can we include a note to that effect in the README?